content/blog/2016/2016-03-02-toulousejam-hackergarten.adoc

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: Jenkins Hackergarten : mercredi 9 mars 2016 à Toulouse
+tags:
+- hackergarten
+- jam
+- meetup
+author: batmat
+---
+
+Vous êtes développeuse ou développeur, vous avez envie de découvrir le projet
+Jenkins de l'intérieur, en bricolant sur un sujet qui vous intéresse, lisez
+la suite !
+
+Mercredi 9 mars, le link:http://www.meetup.com/fr-FR/Toulouse-Jenkins-Area-Meetup/events/228709190/[Toulouse Jenkins Area Meetup]
+organise à Toulouse un Hackergarten Jenkins, occasion idéale pour faire ses
+premiers pas dans la communauté assisté(e) d'un contributeur au projet.
+
+== _Hackergart_ quoi ?
+
+Hackergarten est un mot qui provient de la contraction des mots _Hacker_ et
+_Kindergarten_, ce dernier étant le mot allemand qui désigne en gros l'école
+maternelle.
+
+== Comment ça va se passer ?
+
+En partenariat avec le link:http://www.meetup.com/fr-FR/Toulouse-Java-User-Group/events/228708579/[Toulouse Java User Group]
+(c'est sur ce lien qu'il *faut* s'inscrire), nous nous donnons rendez-vous à
+partir de 18h30 dans les locaux de l'Epitech Toulouse, chacun avec son
+ordinateur (non fourni), et on commence à jouer.
+
+Et c'est bien sûr *gratuit et ouvert à tous*.
+
+Un link:https://trello.com/b/fpT5dJqS/toulousejam-hackergarten-february-2016[tableau trello a été initialisé]
+pour tenter de s'organiser un peu. La liste des choses à faire n'est pas du
+tout figée, et les idées sont les bienvenues.
+
+Les commentaires sont ouverts à tous, et l'accès sera donné à quiconque en fait
+la demande (façon communauté Jenkins :-)).
+
+== Goodies !
+
+Grâce à l'aide de CloudBees, on a pas mal de goodies à offrir : stickers,
+badges, t-shirts et bobble-heads !
+
+=== Les Bobble Heads Jenkins
+image:http://farm3.staticflickr.com/2878/9725573715_fa056b6652_n.jpg[]
```

Comment:
This image URL should be `https://` (or you could import the image into the repo) — otherwise my fussy (secure) browser setup does not load the image.